com.unity.render-pipelines.high-definition/Documentation~/HDR-Output.md

```diff
@@ -0,0 +1,96 @@
+
+The right choice depends on your content, but we suggest to make sure your content is authored to work well without relying on Hue-shift.
+
+#### ACES
+
+Contrary to the neutral version, the ACES tonemapping option has fixed presets. These are meant to target screens with 1000, 2000 and 4000 nits maximum brightness. We suggest to use calibration screen to detect the right preset for the user consuming the content.
+
+![HDR-Output-ACES](C:\Users\franc\Documents\Github\SRP\com.unity.render-pipelines.high-definition\Documentation~\Images\HDR-Output-ACES.png)
+
+##
+
+| **Property**           | **Description**                                              |
+| ---------------------- | ------------------------------------------------------------ |
+| **ACES Preset**        | The tonemapper preset to use. The options are:<br />- ACES 1000 Nits: Curve used to target 1000 nits displays (Default)<br />- ACES 2000 Nits: Curve used to target 2000 nits displays <br />- ACES 4000 Nits: Curve used to target 4000 nits displays |
+| **Detect Paper White** | Whether the paper white value is detected from the data communicated by the display device. We strongly suggest to provide a calibration screen to let the user set this value depending on their screen and viewing experience. |
+| **- Paper White**      | Paper white value (in nits) set when it is not automatically detected from the display. |
+
+#### Custom
+
+Currently not supported by HDR Output. It is possible to decide whether to fallback on Neutral or ACES for outputting to an HDR device.
+
+#### External
+
+Not supported. Mostly because every different HDR screen used to display the content would need a different LUT. It is possible to decide whether to fallback on Neutral or ACES for outputting to an HDR device.
+
+## Control HDR via script
+
+In addition to controlling the above described Volume Component parameters through the camera volume stack, it is also possible to query the [HDROutputSettings](https://docs.unity3d.com/ScriptReference/HDROutputSettings.html). This can be used for example to:
+
+- Query the display brightness limits ([min](https://docs.unity3d.com/ScriptReference/HDROutputSettings-minToneMapLuminance.htmlhttps://docs.unity3d.com/ScriptReference/HDROutputSettings-minToneMapLuminance.html) and [max](https://docs.unity3d.com/ScriptReference/HDROutputSettings-maxToneMapLuminance.html)) and [paper white value](https://docs.unity3d.com/ScriptReference/HDROutputSettings-paperWhiteNits.html).
+- [Request a change in HDR](https://docs.unity3d.com/ScriptReference/HDROutputSettings.RequestHDRModeChange.html), turning it on or off.
+
+## HDR Debug Views
+
+HDRP offers three debug views for HDR rendering. These can be found in **Window > Analysis > Render Pipeline Debugger > Lighting > HDR**
+
+#### Gamut View
+
+Will display on the bottom left of the screen two triangles representing the Rec709 and Rec2020 color gamuts and display what parts of the gamut are covered by the scene.
```

Comment:
Proposal:

"This debug view displays two triangles that indicate which parts of the [Rec709](https://en.wikipedia.org/wiki/Rec._709) and [Rec2020](https://en.wikipedia.org/wiki/Rec._2020) color gamuts the Scene covers."